packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -184,6 +184,9 @@ export class SliceMachineInitProcess {
 			}
 			if (!this.context.repository.exists) {
 				await this.createNewRepository();
+				if (!this.options.starter) {
```

Comment:
⚠️ Technically, this is wrong, and we can and will still have the bug. You assume here you start with a starter with the command the UI gave you, but the bug happens actually when you don't start with the UI. 

Imagine you start for here: https://github.com/prismicio-community/nextjs-starter-prismic-minimal/tree/master
Or from here: https://prismic.io/docs/nextjs-example-projects

As you see, there are no starter option. The condition to check if you need to setup the master locale is if there are documents to push. If you have documents to push, it's what we consider a starter today.

Reply:
Ok, thanks for pointing that out!
Changed

Reply:
Hum so you decide to not check if there are documents to push?

Reply:
ah, ok. I misunderstood your suggestion. Going back to this.